### ISSUE TITLE
macOS support?

### ISSUE THREAD

**Issue by orklann** [google/dart-gl#17]
Can this library work on macOS?

**Comment by jtmcdole** [comment]
I don't have a mac to test it on, but it should work with some caveats.  You need to have the dart-glfw library and glfw3+ for Mac to test it on.

**Comment by boustrophedon** [comment]
I haven't worked on this in a while and John basically rewrote the entire thing (and it was probably a good idea, looking back my code really wasn't good) so I don't know if this applies now, but AFAIK MacOS doesn't support opengl ES on the desktop. Their gl drivers have always been very neglected.

**Comment by orklann** [comment]
Yep, macOS does not support any version of OpenGL ES, so this should fail to build, but I have not tried yet.

**Comment by jtmcdole** [comment]
Looks like you can get it with:
https://moltengl.com/

ANGLE looks like they are working on something, but I don't know their current status.  Chromium has to do something as WebGL 1.0 is ES 2.0.

**Comment by orklann** [comment]
Thanks very much @jtmcdole that looks cool, I will try it later on.

**Comment by jtmcdole** [comment]
Please report back if you have any luck getting this to work on a mac!

**Comment by orklann** [comment]
@jtmcdole I don't have time right now to make this on Mac, I will report it if I can.
